includes/digital-twins-limits.md

```diff
@@ -23,6 +23,7 @@ The table below lists the functional limits of Azure Digital Twins.
 | Routing | Number of routes for a single Azure Digital Twins instance | 6 | Yes |
 | Models | Number of models within a single Azure Digital Twins instance | 10,000 | Yes |
 | Models | Number of models that can be uploaded in a single API call | 250 | No |
+| Models | Maximum size (of JSON Bod in a PUT or PATCH request) of a single model | 1 MB | No |
```

Comment:
Can we change "JSON Bod" to "JSON body" ?

Reply:
Done.